spring-kotlin/aws-ssh-server-manager/src/main/kotlin/dev/santos/awssshservermanager/model/Tenant.kt

```diff
@@ -0,0 +1,14 @@
+package dev.santos.awssshservermanager.model
+
+import javax.persistence.*
+
+@Entity(name = "tenant")
+@Table
+class Tenant(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id:Long=0,
+  val name: String = "",
+  val awsApiKeySsmName: String = "",
+  val awsApiSecretSsmName: String = ""
```

Comment:
Why does it have names? It should be just `awsAccessKeyId` and `awsSecretAccessKey`

Reply:
As agreed, I'm going to store this in the DB directly encrypt it at rest
Thus, I'll rename this as you suggest